lib/app-common.ts

```diff
@@ -218,28 +218,6 @@ function setupBalenaSdkSharedOptions(settings: CliSettings) {
 	});
 }
 
-let BluebirdConfigured = false;
-
-/**
- * Configure Bluebird and assign it as the global promise library.
- * Modules like `stream-to-promise` will otherwise produce native promises,
- * which leads to errors as much of the CLI JavaScript code expects Bluebird
- * promises.
- */
-export function configureBluebird() {
-	if (BluebirdConfigured) {
-		return;
-	}
-	BluebirdConfigured = true;
-	const Bluebird = require('bluebird') as typeof import('bluebird');
```

Comment:
Removing this import means we open up the possibility lazy-loading bluebird and the associated startup time savings